Task/05-calendar

chex0v opened the pull request:
Added calendar widget

Commit:
feature
Added calendar

Commit:
feature
Change background-grey to background-base

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on 05-lection3/03-calendar/calendar.css:
```diff
@@ -0,0 +1,63 @@
+  cursor: pointer;
+}
+
+.c-calendar--item_day {
```
класс немного неправильно написан)
.c-calendar--item_day -> .c-calendar__item--day наверно так?

Commit:
feature
Changed modifier for c-calendar__item. From c-calendar--item_day to c-calendar__item--day

Commit:
Merge branch 'master' into task/05-calendar
